src/js/card.js

```diff
@@ -0,0 +1,98 @@
+import '../css/card.css'
+import { getFolder } from './api';
+import { useEffect, useState } from 'react';
+import noImage from '../assets/images/no-image.svg'
+
+
+function Cards () {
+    const [items,setItems]=useState([]);
+
+    const handleLoad = async()=>{
+        const{ folder } = await getFolder();
+        setItems(folder.links);
+    }
+
+    useEffect(()=>{
+        handleLoad();
+    },[]);
+    
+    return (
+        items.map((item)=><Card key={item.id} items={item}/>)
+    );
+}
+
+
+function Card ({items}) {
+    let image = {};
+    if(items.imageSource===undefined){
+        image = {
+            backgroundImage: `url(${noImage})`,
+        };
+    } else {
+        image = {
+        
+            backgroundImage: `url(${items.imageSource})`,
+        };
+    }
+    let apiDate = new Date(items.createdAt);
+
+    function dateCalculator (apiDate) {
+        let myDate = new Date();
+        let elapsedMinute = Math.floor((myDate - apiDate) / 1000 / 60) ;
+        let elapsedHour = Math.floor((myDate - apiDate) / 1000 / 60 / 60) ;
+        let elapsedDays = Math.floor((myDate - apiDate) / 1000 / 60 / 60 / 24) ;
+        let elapsedMonth = Math.floor((myDate - apiDate) / 1000 / 60 / 60 / 24 / 31) ;
+        let elapsedYear = Math.floor((myDate - apiDate) / 1000 / 60 / 60 / 24 / 31 / 12) ;
+        
+        if (elapsedMinute < 60) {
+            if(elapsedMinute < 2){
+                return (`${elapsedMinute} minute ago`)
+            }
+            return (`${elapsedMinute} minutes ago`)
+        }
+        if (elapsedHour < 24) {
+            if(elapsedHour < 2){
+                return (`${elapsedHour} hour ago`)
+            }
+            return (`${elapsedHour} hours ago`)
+        }
+        if (elapsedDays < 31) {
+            if(elapsedDays < 2) {
+                return (`${elapsedDays} day ago`)
+            }
+            return (`${elapsedDays} days ago`)
+        }
+        if (elapsedMonth < 12) {
+            if (elapsedMonth < 2){
+                return (`${elapsedMonth} month ago`)
+            }
+            return (`${elapsedMonth} months ago`)
+        }
+        if (elapsedYear) {
+            if (elapsedYear < 2){
+                return (`${elapsedYear} year ago`)
+            }
+            return (`${elapsedYear} years ago`)
+        }

```

Comment:
반복되는 구조긴 하지만 여기를 예시로 들자면
```suggestion
        if (elapsedYear) {
            return (`${elapsedYear} year`${elapsedYear > 1 ? 's' : ''} ago`)
        }
```
이런식으로 template literal 을 적극 활용해도 좋을 것 같습니다.